src/track/audio-track.ts

```diff
@@ -1,10 +1,41 @@
-//@flow
 import Track from './track';
 
+enum FlavorAssetTags {
+  AUDIO_ONLY = 'audio_only',
+  AUDIO_DESCRIPTION = 'audio_description'
+}
+
 /**
  * Audio track representation of the player.
  * @classdesc
  */
-class AudioTrack extends Track {}
+class AudioTrack extends Track {
+  /**
+   * @member - is audio asset has audio description tag
+   * @type {boolean|undefined}
+   */
+  public isAudioDescription?: boolean;
+}
+
+export function audioDescriptionTrackHandler(tracks: AudioTrack[], audioFlavors?: Array<any>): void {
+  if (tracks?.length) {
```

Comment:
if is not set or flavors.length is 0 i think we can just check it once at the top of the function and return tracks as is, instead of checking audioFlavors?[index]? every time